cross/emby/Makefile

```diff
@@ -0,0 +1,27 @@
+PKG_NAME = emby
+PKG_VERS = 3.0.5597.1
+PKG_EXT = zip
+PKG_DIST_NAME = MediaBrowser-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/MediaBrowser/MediaBrowser.Releases/raw/master/Server
+PKG_DIR = emby
+
+DEPENDS = 
+
+HOMEPAGE = http://emby.media/
+COMMENT  = Emby streams everything Your personal videos, music, photos, and live television in the palm of your hand.
+LICENSE  =
+
+CONFIGURE_TARGET = nop
+COMPILE_TARGET = nop
+INSTALL_TARGET = myInstall
+
+EXTRACT_TARGET = myExtract
+
+include ../../mk/spksrc.cross-cc.mk
+
+myInstall:
+	mkdir -p $(STAGING_INSTALL_PREFIX)/share/$(PKG_DIR)
+	tar -cf - -C $(WORK_DIR)/$(PKG_DIR) . | tar -xf - -C $(STAGING_INSTALL_PREFIX)/share/$(PKG_DIR)
+
+myExtract:
+	unzip ../../distrib/$(PKG_DIST_NAME) -d $(WORK_DIR)/$(PKG_DIR)
```

Comment:
If the reason for this is an incorrect `EXTRACT_PATH`, then may I suggest to specify `EXTRACT_PATH = $(WORK_DIR)/$(PKG_DIR)` instead of the current `EXTRACT_TARGET` override?
 
See e.g. [selfoss](https://github.com/SynoCommunity/spksrc/blob/03f583a3db38b80021aa9e7c5af641ac2b9fdf0b/cross/selfoss/Makefile#L14)